### ISSUE TITLE
AB Role in TAG Appointment

### ISSUE THREAD

**Issue by martinthomson** [w3c/w3process#810]
Decisions about TAG appointments are currently [ratified by a two-thirds majority of both the TAG and the AB](https://www.w3.org/2023/Process-20231103/#TAG-appointments).  I do not see a reason for the AB to be involved in this decision at that level.

For boards and groups operating at the level of the TAG, an appointment process is a good way to ensure that there is sufficient diversity of members available.  Particularly when elections are used to fill seats, the process of filling seats can leave gaps in expertise or other qualities that would diminish the capacity of the group to function effectively.

A more regular process would be to give the group itself the ability to fill a small number of positions.  For some reason, the W3C process gives this responsibility to the Team.  That is somewhat irregular, but not really the subject of this issue.

The decision by the Team then needs to be approved by the TAG with a two-thirds majority, which is entirely appropriate.  If the TAG believes that they cannot work with an appointed person, or the appointments do not address a diversity shortfall, then the TAG needs to be able to have a say.

The AB is given the same approval power, something that seems poorly justified.  Obviously, the AB and TAG frequently work together, but while the two functions are complementary, the AB has no significant stake in the composition of the TAG or the technical work that the TAG does.  The function of the AB is to advise the team, yes, but that is generally limited to procedural and operational matters.

Giving the AB input into TAG appointments at the same level as the TAG is unnecessary.  I see no substantive reason that the AB needs a role that is significantly greater than any other consortium member.

If the goal of having the AB vote on appointments is to provide better accountability to the members of the organization, the formal objection process is a more appropriate way to ensure that membership has a way to object to particular appointments.  That option is available to the AB just like any community member.

I propose that the process be amended to remove the AB from this process.

**Comment by frivoal** [comment]
Just tying things together, not arguing one way or the other. The AB's participation in the ratification came from:
* issue: https://github.com/w3c/w3process/issues/715
* AB discussion: https://www.w3.org/2023/03/16-ab-minutes.html#t07
* PR: https://github.com/w3c/w3process/pull/718

**Comment by frivoal** [comment]
I agree with the issue as raised, and am unconvinced that the AB having a say in this is necessary or helpful, at least in the current form.

I do think that having the TAG ratify the Team's appointment is an effective way to ensure that the Team did have the necessary consultation with the TAG, so I think that's important to keep, but under the current format, I'm not sure the AB's involvement does anything meaningful. My support in the past for what has now become the Process came not so much from including the AB, as from keeping the TAG in the loop. For the AB's inclusion, I was, and remain, unconvinced.

Maybe, if we want to preserve both the ability of the TAG to push back in case the Team's consultation is insufficient, AND have some mechanism to prevent the TAG from being self-serving, a possible process could be:
* The Team consults with various people, TAG included (unchanged)
* The Team proposes the appointments (unchanged)
* The TAG (alone) ratifies those appointments. If they do, process ends here.
* If the TAG fails to ratify the Teams appointment, then we go to the AB to settle the dispute. After looking into it, they can side with the TAG and block the appointments if they feel the Team indeed didn't do an appropriate job, or ratify the seats over the TAG's wishes if they find that it is the TAG that is being unreasonable / self serving…

**Comment by martinthomson** [comment]
That sounds like a reasonable process, though I'd like to poke on the conflict resolution piece a little.

I don't see a particular problem in the case of a failed TAG approval if the Team wants to go back and propose an alternative option.  Maybe the TAG says "no" and then the Team and TAG talk and find out what resulted in that outcome.  Maybe they can agree on a do-over.

Obviously, if there is a real disagreement, having an independent arbiter is useful.  The AB seems like a reasonable choice for serving that function.

The other option on the table is not making appointments.  It is implied in policy, but not explicit, that the Team has no choice in this.  That is, the Team MUST make three and exactly three appointments.  I'm not sure that I like that.  Not just because a looser "MAY appoint up to three" would be a better way to ensure that the TAG membership is accountable to membership, but because it creates another alternative outcome in the case of conflict.  (Maybe that's a separate issue though, as the decision to not appoint is also a decision that would need ratification/approval.)

**Comment by frivoal** [comment]
>  It is implied in policy, but not explicit, that the Team has no choice in this. That is, the Team MUST make three and exactly three appointments.

I'm not sure. I could easily read the text as a SHOULD as well.

https://www.w3.org/2023/Process-20231103/#TAG-appointments

**Comment by martinthomson** [comment]
Wonderful.  The use of "SHOULD" there definitely makes it unclear, but my initial read was that those uses were to recommend how the Team was expected to undertake the job, not that the appointments were discretionary.  To the new issue page!

**Comment by cwilso** [comment]
I disagree.  There was a thorough discussion of this in the AB at the request of the Process CG (Florian linked to the minutes above), and there was a clear reasoning - to avoid insiderism on the TAG.  Although it might be appropriate to review this in a few years, the TAG is only now considering the first appointments it is involved in.  I don't think it is appropriate to consider jettisoning this at this point, as we don't have any new information about how this might play out.

At the same time, I would point out it's incorrect to say the AB has the same level of input into TAG appointments as the TAG itself.  The Process is that the Team works together with the TAG to determine appointees - that is then ratified by the TAG and the AB.

Likewise I would claim it's incorrect that "the AB has no significant stake in the composition of the TAG or the technical work that the TAG does... [the AB's advice] is generally limited to procedural and operational matters."  The AB has a broad charter, and with the addition of the Board, operational matters are largely out of the AB's purview; it is expected that the AB will be involved with the technical agenda of the W3C (at least, I've heard that explicitly started by Board members).  But at any rate, this was meant as a high-level guardrail against insiderism, and I don't understand what has changed since that decision was made.

**Comment by frivoal** [comment]
> there was a clear reasoning - to avoid insiderism on the TAG

I'm unsure how well this works. With the current ratification system, the TAG would have no difficulty in rejecting outsiders, which largely boils down to a similar thing.

>  I don't think it is appropriate to consider jettisoning this at this point, as we don't have any new information about how this might play out.

It is true that we have no experience with this new system, and we're likely to learn from using it. I'm not in a particular rush, and I don't think it is especially likely to cause problems. I'm just unconvinced that it is apt at solving the kind of problem it was intended for.

**Comment by martinthomson** [comment]
I don't see any particular urgency either.  With #809, which would let the process run longer, I would suggest replacing the AB vote with a membership vote instead.

I agree with @frivoal as far as the "insiderism" thing.

**Comment by dwsinger** [comment]
I agree, I see neither urgency to change nor forceful reasons for the current text. If the AB did not have a formal vote, it still has influence: talk to the TAG, talk to the team. We could even say that the AB must be advised at the same time as the TAG is scheduled to approve (as at that point the appointments might not be public).

**Comment by plehegar** [comment]
As a side, we'll need a post-mortem on the Process once the TAG appointments are announced. I expect the Team will input to provide on how the Process worked for them.

**Comment by nigelmegitt** [comment]
@plehegar flagging use of phrase "post-mortem" - many folk don't like this for various reasons.

**Comment by TallTed** [comment]
@nigelmegitt — "[Flagging use of [a] phrase](https://github.com/w3c/w3process/issues/810#issuecomment-1866044408)" without offering alternatives is not nearly as effective nor helpful as offering *some* alternative, even if that then spawns a bike-shedding cycle.

@plehegar — In the spirit of progress, a couple of alternatives that come to my mind, that might be put to good use when you open a followup issue to track [your wish](https://github.com/w3c/w3process/issues/810#issuecomment-1865008351), are "review of lessons learned" and "retrospective review".

**Comment by brentzundel** [comment]
@hober will take an action here

**Comment by brentzundel** [comment]
The AB feedback at this point is that the AB does not feel they should be involved in ratifying TAG appointments.

**Comment by frivoal** [comment]
Does this also imply that the AB thinks it should not be involved in resolving/escalating disagreements between the Team and TAG when they get stuck on ratifying appointments, or is that a separate consideration that the AB has not opined on?

**Comment by brentzundel** [comment]
> Does this also imply that the AB thinks it should not be involved in resolving/escalating disagreements between the Team and TAG when they get stuck on ratifying appointments, or is that a separate consideration that the AB has not opined on?

I would say that is a separate consideration. I believe the AB would be happy to advise the Team and TAG if they get stuck on ratifying appointments.

**Comment by frivoal** [comment]
> I would say that is a separate consideration.

Yes and no. While I think the current set-up does not achieve that goal, it was my impression that the original intent behind the involvement of the AB in the TAG nomination process was to be able to intervene if either the TAG or the Team was being unreasonable.

So, since the current approach doesn't do that, it is kind of a separate question. But since it was trying to do that, it isn't really.

> I believe the AB would be happy to advise the Team and TAG if they get stuck on ratifying appointments.

That's certainly an option. But the underlying question is whether the AB should have some _decision_ power about this. For instance, a possible design is:
* The Team proposes, the TAG validates (as today). If the TAG does validate, end of story, and the AB doesn't get involved.
* If the TAG rejects the Team's proposal, they can talk it out, and try again, as many times as necessary to try and reach a compromise. However, either the TAG or the Team can escalate if they thing the other body is behaving unreasonably, in which case it goes to the AB. Then, the AB could decide, if they think the TAG is unreasonable, to approve the Team's appointment anyway, or if they think the Team is unreasonable, to let the TAG handle the appointment themselves this time.

I'm not saying this (or something like this) is necessarily what we should do, but it is what this question is about. If the AB has concluded we shouldn't do that, that's answered, and we can move on with that conclusion. If that hasn't been discussed, then I think it may be premature to move forward with edits.

**Comment by martinthomson** [comment]
That depends on a presumption of one or other party (TAG or Team) being somehow “wrong”. That seems like a mistake in characterization. The situation we are talking about is a straightforward disagreement. The only thing you need to consider is “what do you do of the Team proposes an appointment and the TAG rejects that?”

That is an outcome that has happened. The world did not end. The TAG was “short” for a period, that’s all. I suggest that maybe we not try to “fix” that problem. The AB can definitely advise or help (an FO would guarantee their involvement, which suffices in my view) but they don’t need to.

**Comment by frivoal** [comment]
> That depends on a presumption of one or other party (TAG or Team) being somehow “wrong”.

The assumption behind the initial attempt to involve the AB was not that either party would necessarily be wrong in case of disagreement. The intent was to have a stop gap _in case_ the disagreement is due to either (or both) of them being unreasonable. This is not a presumption that they will be, but a backstop in case they ever are.

> an FO would guarantee [the AB's] involvement, which suffices in my view

It would. But then that would go to the TAG as well as to the AB. As I recall, a goal of having a special procedure was to avoid that.

----

Again, I am not especially advocating for any particular solution, just pointing out mitigating self-dealing or deadlocks was the whole reason for this existing, and that it seems premature to wrap this up if this has not been considered. If it has been, great, but the report from @brentzundel about AB discussions does not make this evident.

**Comment by frivoal** [comment]
After discussion in the Process CG, it was further explained that the AB did indeed discuss these aspects, and concluded that the AB should be removed from the ratification of TAG appointees.
